### PR TITLE
[GL] GetState sentences

### PR DESCRIPTION
**Pull request by svillar**

### CODE REVIEW

**Comment by svillar** [comment]
This requires #1748 as I'm using `<nome>` instead of `<name>`

File: sentences/gl/homeassistant_HassGetState.yaml [hunk 1]
```diff
@@ -1,4 +1,36 @@
 language: gl
 intents:
   HassGetState:
-    data: []
+    data:
+      - sentences:
+          - "<cal> é o estado [do|da] <nome> [<area>]"
+	  - "en que estado <esta> <nome> [<area>]"
```
Comment (cibernox) [inline sentences/gl/homeassistant_HassGetState.yaml:7]: Wrong indent?

Comment (svillar) [reply sentences/gl/homeassistant_HassGetState.yaml]: tab -> whitespace. Done

Comment (svillar) [reply sentences/gl/homeassistant_HassGetState.yaml]: Added also responses and tests

**Pull request merged**